### PR TITLE
Feature/upc service

### PR DESCRIPTION
**Pull request by jvillarreal-kpmg**
With this branch you get:
- in cases where fda service didn't provide a product from barcode, it gets product info from factual service and do a keyword search from result
- parses upc code from fda result, and shows a product image from factual result data when found
- shows title with ellipses for details screen

Some important notes
- I got the factual account verified and they now increased the free service to up to 10k requests/day for most data.
- For a quick check on their product catalog go to:
  http://www.factual.com/products/cpg

### CODE REVIEW

**Comment by jvillarreal-kpmg** [comment]
#12 Is related

File: www/modules/recalls/controllers/SearchCtrl.js [hunk 1]
```diff
@@ -65,21 +65,36 @@ main.controller('SearchCtrl', function (
...
-                //TODO: can we get product info from barcode and do a keyword search?
+
+                // get product info from barcode and do a keyword search
+                factualUpcService.getData(barcode).success(function (result) {
+                    var products = factualUpcService.getProducts(result);
+                    var product;
+                    if (products && products.length) {
+                        product = products[0];
+                        foodEnforcementService.getRecallsByKeyword(product.name)
+                            .success(function (result) {
```
Comment (jardilio-kpmg) [inline www/modules/recalls/controllers/SearchCtrl.js]: There is no need for success and error handlers here, just simply call the getResultsByKeywords method

File: WebStart.NET - Client Code.csproj [hunk 2]
```diff
@@ -0,0 +1,714 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (jardilio-kpmg) [inline WebStart.NET - Client Code.csproj]: please delete this file, should have been removed in prior commits upstream

**Comment by jvillarreal-kpmg** [comment]
Fixed.

**Comment by jardilio-kpmg** [comment]
Fixed #12

**Pull request merged**